pkg/client/cache/listers_core.go

```diff
@@ -71,3 +73,72 @@ func (s storePodsNamespacer) Get(name string) (*api.Pod, error) {
 	}
 	return obj.(*api.Pod), nil
 }
+
+/*****************************************************************************/
```

Comment:
Unusual, we don't do this.  Remove.  Add nice godoc to StoreToReplicationControllerLister instead.